### ISSUE TITLE
HTTPS DNS records

### ISSUE THREAD

**Issue by arjenpdevries** [dir2019/dir2019.github.io#28]
With https, only www.dir2019.nl works, but not dir2019.nl

It is the DNS (but I do not know exactly what to do 🤓).

**Comment by graus** [comment]
Thanks, good find. It unfortunately seems like it is a Github pages thing: 

> GitHub doesn't currently support creating a certificate that covers both your root domain and your `www` subdomain. We only generate a certificate for the exact domain that you specify in the custom domain input box. 
https://github.community/t5/GitHub-Pages/Does-GitHub-Pages-Support-HTTPS-for-www-and-subdomains/m-p/7202#M495

**Comment by arjenpdevries** [comment]
Ah, but if I go to https://informagus.nl/ it reroutes to https://www.informagus.nl/ and both links work. So maybe you need to remove www from the CNAME file and the problem is resolved?

**Comment by graus** [comment]
Let us try!

**Comment by arjenpdevries** [comment]
Hmmm, wait, I just noticed that my CNAME file does say www.informagus.nl...

**Comment by graus** [comment]
I mean, looking forward to receiving your PR, @arjenpdevries 😬

**Comment by arjenpdevries** [comment]
I'll see if I can find the thing that makes it work for me and not for dir2019.nl but not sure...

**Comment by graus** [comment]
Hmm, perhaps I can add a redirect someplace else? (with the domain hoster?)

**Comment by arjenpdevries** [comment]
Yes, compare the output of `dig informagus.nl` to `dig dir2019.nl`.

Your A record points to 188.93.150.76, which is not github!
```
whois 188.93.150.76
```

I set the records to point to the github addresses.
```
whois 185.199.110.153
```

See: https://help.github.com/en/articles/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain

**Comment by arjenpdevries** [comment]
(Sent you email with exact settings in my provider - at the graus.nu email)

**Comment by graus** [comment]
Thanks! Got it. I'll take a look
